lib/presentation/organisms/intro/tips_organism.dart

```diff
@@ -37,6 +38,24 @@ class TipsOrganism extends StatelessWidget {
             translate: false,
           ),
           const SeparatorAtom(variant: SeparatorVariant.farApart),
+          Center(
+              child: ElevatedButton(
+                  onPressed: () async {
+                    bool filter =
+                        await FlutterDnd.getCurrentInterruptionFilter() ==
+                            FlutterDnd.INTERRUPTION_FILTER_PRIORITY;
+                    if (await FlutterDnd.isNotificationPolicyAccessGranted ??
+                        false) {
+                      await FlutterDnd.setInterruptionFilter(
+                        filter
+                            ? FlutterDnd.INTERRUPTION_FILTER_ALL
+                            : FlutterDnd.INTERRUPTION_FILTER_PRIORITY,
+                      ); // Turn on DND - All notifications are suppressed.
+                    } else {
+                      FlutterDnd.gotoPolicySettings();
+                    }
+                  },
+                  child: Text("Toggle DND"))),
```

Comment:
Code to add button with DND Toggle Functionality

Reply:
Since there are unsupported platforms like iOS the button should have different behavior.

In case of an unsupported platform, it should change to text like
"We recommend to toggle DND"